FileGopherServer.cpp

```diff
@@ -242,7 +247,7 @@ void FileGopherServer::start(int n_connections)
         int n = read(newsocket_fd, buffer, BUFFER_LEN);
         if (n < 0)
             error("ERROR reading from socket");
-        int size = get_contents(buffer, n);
+        int size = get_contents(buffer, n, false);
```

Comment:
Shouldn't the third param be `show_hidden_files`

Reply:
Sorry i took the default value there, my bad !